### ISSUE TITLE
AdKiller didnt work anymore ?

### ISSUE THREAD

**Issue by SnowAxe** [SecUpwN/Spotify-AdKiller#116]
Hello,

10 days ago i have reinstall my Kali Linux to make a dual boot with Windows 10.
Today i tried AdKiller because im bored of YouTube, i used these command

sudo apt-get install x11-utils pulseaudio-utils libnotify-bin xdotool vlc
cd /Documents
git clone https://github.com/SecUpwN/Spotify-AdKiller.git
cd Spotify-AdKiller
chmod +x ./install.sh
./install.sh

Then i have install Spotify i don't know if i can remember this but when i start spotify by the command line "spotify" the name of the app is Spotify Free but im not sure before it was just Spotify.

EDIT: im on Kali
Of course, now i have advert ...

**Comment by fassn** [comment]
Important thing I just figured out: don't install Spotify using snap if that is the case. I'm on Ubuntu, and realized this tool isn't working with a Spotify installed from snap. I had to uninstall it, and reinstall through apt in my case.

**Comment by Niru2169** [comment]
Yes. And apart from that, you mustn't start spotify from command line
Launch the Spotify (Adkiller).desktop
The .desktop file executes spotifywrapper.sh

**Comment by Niru2169** [comment]
Also, lock Spotify and prevent it from updating.
That might help too

**Comment by apes0** [comment]
> you mustn't start spotify from command line

huh, seems to work fine here!

**Comment by AnkushSinghGandhi** [comment]
> Important thing I just figured out: don't install Spotify using snap if that is the case. I'm on Ubuntu, and realized this tool isn't working with a Spotify installed from snap. I had to uninstall it, and reinstall through apt in my case.

do i need to install adkiller again after installing spotify through apt
